src/App.js

```diff
@@ -45,6 +45,7 @@ const {modalDisplay, setModalDisplay}=useDataValues();
         <Route path="/mock" element={<Mockman/>}/>
         <Route path="/playlist-page/:id" element={<PlayListPage/>}/>
         <Route path="*" element={<Page404/>}/>
```

Comment:
appreciation - good that you've added 404 page

Reply:
Thanks for appreciation